Tasks/DownloadSecureFileV1/Tests/L0.ts

```diff
@@ -4,22 +4,67 @@ import path = require('path');
 import * as ttm from 'azure-pipelines-task-lib/mock-test';
 
 describe('DownloadSecureFile Suite', function () {
-    this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
     before(() => {
     });
 
     after(() => {
     });
 
     it('Defaults: download secure file', (done: MochaDone) => {
-        this.timeout(1000);
+        this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
 
         let tp: string = path.join(__dirname, 'L0SecureFile.js');
         let tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
 
         tr.run();
 
         assert(tr.stderr.length === 0, 'should not have written to stderr');
+        assert(tr.stdOutContained('##vso[task.debug]Mock SecureFileHelpers retry count set to: 5'), 'task should have used default retry count of 5');
+        assert(tr.succeeded, 'task should have succeeded');
+
+        done();
```

Comment:
You actually don't have to call or pass `done` in any of these.  If you pass it you have to call it, but if you don't pass it it's fine.

`done` is for getting the test to wait until async callbacks have completed, which doesn't apply here.